client/components/data/query-activity-log/index.jsx

```diff
@@ -1,34 +1,43 @@
+/** @format */
 /**
  * External dependencies
  */
-import { Component, PropTypes } from 'react';
+import { PureComponent } from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { activityLogRequest as activityLogRequestAction } from 'state/activity-log/actions';
 
-class QueryActivityLog extends Component {
+class QueryActivityLog extends PureComponent {
 	static propTypes = {
 		siteId: PropTypes.number,
+
+		// Unix millisecond timestamps ( moment.valueOf() )
+		dateEnd: PropTypes.number,
+		dateStart: PropTypes.number,
+
+		// Number of results
+		number: PropTypes.number,
```

Comment:
That's probably a good idea. `20` is what the API uses right now, so I'll put that in here for consistency (and to ensure it doesn't change on us).